docker-compose.yaml

```diff
@@ -27,6 +27,29 @@ services:
       timeout: 2s
       retries: 10
 
+  # Redis - caching + rate limiting on BE
+  redis:
+    image: redis
+    restart: always
+    ports:
+      - 6379:6379
+    healthcheck:
+      test: ["CMD-SHELL", "redis-cli ping | grep PONG"]
+      interval: 2s
+      timeout: 2s
+      retries: 10
+    command: redis-server /usr/local/etc/redis/redis.conf
+    volumes:
+      - ./redis.conf:/usr/local/etc/redis/redis.conf
+      - redis:/data
+  # insights host - redis:6379
+  redis-insights:
+    image: redislabs/redisinsight:latest
+    ports:
+      - 8001:8001
+    volumes:
+      - redis:/data
```

Comment:
do you think it's necessary to define a volume here? seems like for a test setup it's fine if the data goes down with the container, especially bc we don't use this redis as long-term storage

Reply:
Fair point. Removed 👍